components/nav/OrderStepper.tsx

```diff
@@ -1,35 +1,46 @@
 'use client'
 import React, { useEffect, useState } from 'react';
-import { Stepper, Step, StepLabel, Typography, Box } from '@mui/material';
+import { Stepper, Step, StepLabel, Box, Alert } from '@mui/material';
 import SelectServiciesStep from '../step/SelectServiciesStep';
 import SelectPatientStep from '../step/SelectPatientStep';
 import BlueButton from '../buttons/BlueButton';
 import { useOrderStore } from '@/src/store';
+import GreyButton from '../buttons/GreyButton';
+import OrderPreviewStep from '../step/OrderPreviewStep';
 
 
-const steps = ['Seleccionar Estudios', 'Completar Datos Cliente', 'Revisar y Confirmar'];
+
+const steps = ['Seleccionar Estudios', 'Seleccionar Cliente', 'Revisar y Confirmar'];
 
 export default function OrderStepper() {
   const selectedLabItems = useOrderStore( (state) => state.selectedLabItems)
+  const selectedClient = useOrderStore( (state) => state.selectedClient)
   const [isDisabled, setIsDisabled] = useState(true);
+  const activeStep = useOrderStore((state) => state.activeStep);
+  const setActiveStep = useOrderStore((state) => state.setActiveStep);
+  const resetOrder = useOrderStore((state) => state.resetOrder);
 
+  // Para desactivar el boton siguiente.
   useEffect(() => {
-    setIsDisabled(selectedLabItems.length === 0); 
-  }, [selectedLabItems])
+    if (activeStep === 1) { 
```

Comment:
I prefer that we use constants. For example, we can define them at the top of the file:
```
const STEP_1 = 1;  
if (activeStep === STEP_1) {  
  ...  
}  
```

instead of:

```
if (activeStep === 1) {  
  ...  
}  
```
This is to improve code readability